core/src/main/java/org/apache/iceberg/deletes/BitmapPositionDeleteIndex.java

```diff
@@ -39,7 +39,12 @@ public void delete(long posStart, long posEnd) {
   }
 
   @Override
-  public boolean deleted(long position) {
+  public boolean isDeleted(long position) {
```

Comment:
why changing the name?

Reply:
It is suggested by @rdblue. Make sense to me as well.